runtime/src/snapshot_utils.rs

```diff
@@ -395,6 +398,38 @@ pub fn archive_snapshot_package(
         ("duration_ms", timer.as_ms(), i64),
         ("size", metadata.len(), i64)
     );
+
+    if let Some(snapshot_type) = snapshot_package.get_snapshot_type() {
+        if snapshot_type.is_full_snapshot() {
+            info!("Create snapshot local file in {:?}", tar_dir);
+            create_local_snapshot_file(tar_dir)?;
+        }
+    }
+    Ok(())
+}
+
+pub fn has_local_snapshot_file<P: AsRef<Path>>(snapshot_archives_dir: P) -> bool {
+    snapshot_archives_dir
+        .as_ref()
+        .join(LOCAL_SNAPSHOT_FILE_NAME)
+        .exists()
+}
+
+pub fn create_local_snapshot_file<P: AsRef<Path>>(snapshot_archives_dir: P) -> Result<()> {
+    let _local_file = File::create(
+        snapshot_archives_dir
+            .as_ref()
+            .join(LOCAL_SNAPSHOT_FILE_NAME),
+    )?;
+    Ok(())
+}
+
+pub fn delete_local_snapshot_file<P: AsRef<Path>>(snapshot_archives_dir: P) -> Result<()> {
+    let _ = fs::remove_file(
+        snapshot_archives_dir
+            .as_ref()
+            .join(LOCAL_SNAPSHOT_FILE_NAME),
+    );
```

Comment:
Instead of swallowing _all_ the errors, I think only [`NotFound`](https://doc.rust-lang.org/std/io/enum.ErrorKind.html#variant.NotFound) should be swallowed.